tests/test_json_regex.py

```diff
@@ -0,0 +1,64 @@
+import unittest
```

Comment:
I presume we would like to name it `tests/utils/test_path_mapping.py`, so that it matches the module structure inside the package and starts with test, which is the default pattern used by unittest library.

I added some tests as reference in 5b6098320cd45ba0dfe5f32fc3a1e0c6444100e1

Reply:
pulled

Reply:
Why do we put test cases under `/utils`?